### PR TITLE
web: codemod `ExtensionCard` component to CSS modules

### PR DESCRIPTION
**Pull request by valerybugakov**
## Context

This PR results from the [global-css-to-css-modules](https://github.com/sourcegraph/codemod/tree/main/src/transforms/globalCssToCssModule) codemod application with some manual fixes, caused by the usage of global CSS classes tied to a React component in other parts of the codebase: e.g., `.sidebar` class used outside of the `<Sidebar />` component.

## Changes

- The `ExtensionCard` stylesheet is converted to the CSS module.

### CODE REVIEW

**Review by umpox — APPROVED** [review]
🔥

File: client/web/src/extensions/ExtensionCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    }
+
+    // 300ms delay. We could use `animation-delay`, but the
+    // percentages are cleaner with 2s vs 1.7s.
+    15% {
+        box-shadow: none;
+    }
+
+    40% {
+        box-shadow: 0 0 0 6px var(--success-4);
```
Comment (valerybugakov) [inline client/web/src/extensions/ExtensionCard.module.scss:22]: @tjkandala, please take a look at this change. Previously we used the `rgba` custom color with opacity. `var(--success-4)` looks like the closest match that we can reuse without relying on the SCSS variable here.

Comment (tjkandala) [reply client/web/src/extensions/ExtensionCard.module.scss:22]: This looks great

**Review by tjkandala — APPROVED** [review]

File: client/web/src/extensions/ExtensionCard.module.scss (hunk 1 above)
Comment (valerybugakov) [reply client/web/src/extensions/ExtensionCard.module.scss:22]: 🙌

**Pull request merged**